content/blog/open-colors/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Open Colors
+date: 2020-11-09
+author: Lipis
+link: https://twitter.com/lipis
+---
+
```

Comment:
```suggestion

(Part of a series where we pick apart various libraries and projects we use on Excalidraw.)
```

Reply:
Best if we also come up with a title prefix as discussed.

Reply:
I added that as a note in frontmatter...